R/tm_outliers.R

```diff
@@ -11,17 +11,37 @@
 #' Specifies variable(s) to be analyzed for outliers.
 #' @param categorical_var (`data_extract_spec` or `list` of multiple `data_extract_spec`) optional,
 #' specifies the categorical variable(s) to split the selected outlier variables on.
+#' @param table_decorator (`list` of `teal_transform_module`) optional,
+#' decorator for the table.
+#' @param boxplot_decorator (`list` of `teal_transform_module`) optional,
+#' decorator for the box plot.
+#' @param violin_decorator (`list` of `teal_transform_module`) optional,
+#' decorator for the violin plot.
+#' @param density_decorator (`list` of `teal_transform_module`) optional,
+#' decorator for the density plot.
+#' @param cum_dist_decorator (`list` of `teal_transform_module`) optional,
+#' decorator for the cumulative distribution plot.
 #'
 #' @templateVar ggnames "Boxplot","Density Plot","Cumulative Distribution Plot"
 #' @template ggplot2_args_multi
 #'
 #' @inherit shared_params return
 #'
+#' @section Decorating `tm_outliers`:
+#'
+#' This module creates below objects that can be modified with decorators:
```

Comment:
The phrasing seems off

```suggestion
#' This module generates the following objects, which can be modified in place using decorators:
```